.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "demos/axelar-token/lib/forge-std"]
 	path = demos/axelar-token/lib/forge-std
 	url = https://github.com/foundry-rs/forge-std
+[submodule "contracts/lib/ExcessivelySafeCall"]
```

Comment:
Why using git submodule?

Reply:
I think this is the default way to install forge modules.